server/app/services/customer_db_service.py

```
@@ -0,0 +1,472 @@
+            
+            if not customer:
+                logger.warning(f"No customer found for phone {phone_number} in org {organization_id}")
+                return False
+            
+            customer.call_attempts += 1
+            customer.last_call_attempt_at = datetime.now(timezone.utc)
+            
+            await db.commit()
+            
+            logger.info(f"Incremented call attempts to {customer.call_attempts} for customer {phone_number}")
+            return True
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)